src/Types/TxConstraints.purs

```diff
@@ -320,6 +323,9 @@ mustSatisfyAnyOf =
     >>> MustSatisfyAnyOf
     >>> singleton
 
+mustNotBeValid :: forall (i :: Type) (o :: Type). TxConstraints i o
```

Comment:
We really need a doc comment for this, I don't think most users are familiar with the purpose of `isValid` and why you'd ever want to set it to `false`. I saw the note in the changelog but it should also be here